docs/dev/swagger.yaml

```diff
@@ -0,0 +1,290 @@
+
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: Open Monograph Press API
+  description: API to interact with Open Monograph Press
+  contact:
+    name: Public Knowledge Project
+    url: 'http://pkp.sfu.ca'
+produces:
+  $ref: '../../lib/pkp/docs/dev/swagger/produces.yaml'
+securityDefinitions:
+  $ref: '../../lib/pkp/docs/dev/swagger/securityDefinitions.yaml'
+security:
+  $ref: '../../lib/pkp/docs/dev/swagger/security.yaml'
+host: localhost:8000
+basePath: /index.php/publicknowledge/api/v1
+schemes:
+  $ref: '../../lib/pkp/docs/dev/swagger/schemes.yaml'
+tags:
+  - name: "Submissions"
+  - name: "Series"
+  - name: "Categories"
+  - name: "Users"
+    description: Only admins, journal editors, and subeditors can access these endpoints.
+  - name: "Backend"
+    description: "Private endpoints for OJS UI."
+    
+definitions:
+  $ref: '../../lib/pkp/docs/dev/swagger/sharedDefinitions.yaml'
+  Submission:
+    type: object
+    properties:
+      $ref: '../../lib/pkp/docs/dev/swagger/sharedSubmissionProperties.yaml'
+      chapters:
+        type: array
+        items:
+          $ref: '#/definitions/Chapter'
+  SubmissionSummary:
+    type: object
+    properties:
+      $ref: '../../lib/pkp/docs/dev/swagger/sharedSubmissionSummaryProperties.yaml'
```

Comment:
Let's call these files just `definitions.yaml`, `Submission.yaml` and `SubmissionSummary.yaml`. Their position in the `lib/pkp` directory tells us that they are shared definitions.

Reply:
no problem. I will rename the files.